web/app/scripts/details/details-tabs-partial.html

```diff
@@ -4,19 +4,19 @@
          ng-init="data = ctl.record.data[definition.propertyName]; properties = ctl.sortedProperties(definition.properties)">
 
         <driver-details-single
+            ng-if="!definition.multiple"
```

Comment:
This was just for readability. I felt like putting the ng-if at the end was burying the lede.